static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -130,30 +130,31 @@ function BaseDraggableTabList({
           ref={tabListRef}
         >
           {persistentTabs.map(item => (
-            <Reorder.Item
-              key={item.key}
-              value={item}
-              style={{display: 'flex', flexDirection: 'row'}}
-              as="div"
-            >
-              <Tab
+            <Fragment key={item.key}>
+              <Reorder.Item
                 key={item.key}
-                item={item}
-                state={state}
-                orientation={orientation}
-                overflowing={
-                  orientation === 'horizontal' && overflowTabs.includes(item.key)
-                }
-                ref={element => (tabItemsRef.current[item.key] = element)}
-                variant={tabVariant}
-              />
-
+                value={item}
+                style={{display: 'flex', flexDirection: 'row'}}
+                as="div"
+              >
+                <Tab
+                  key={item.key}
+                  item={item}
+                  state={state}
+                  orientation={orientation}
+                  overflowing={
+                    orientation === 'horizontal' && overflowTabs.includes(item.key)
+                  }
+                  ref={element => (tabItemsRef.current[item.key] = element)}
+                  variant={tabVariant}
+                />
+              </Reorder.Item>
               {(state.selectedKey === 'temporary-tab' ||
                 (state.selectedKey !== item.key &&
                   state.collection.getKeyAfter(item.key) !== state.selectedKey)) && (
                 <TabDivider />
               )}
-            </Reorder.Item>
+            </Fragment>
```

Comment:
Separate the `<TabDivider/>` from the `<Reorder.Item/>` component. This previously allowed users to drag the tab from the tab divider, which is visually separate from the tab.